src/Ownership/OwnershipRepository.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Ownership;
+
+use Broadway\EventHandling\EventBus;
+use Broadway\EventSourcing\AggregateFactory\PublicConstructorAggregateFactory;
+use Broadway\EventSourcing\EventSourcingRepository;
+use Broadway\EventStore\EventStore;
+
+final class OwnershipRepository extends EventSourcingRepository
+{
+    public function __construct(
+        EventStore $eventStore,
+        EventBus $eventBus,
+        array $eventStreamDecorators = []
+    ) {
+        parent::__construct(
+            $eventStore,
+            $eventBus,
+            Ownership::class,
+            new PublicConstructorAggregateFactory(),
+            $eventStreamDecorators
+        );
+    }
+
+    public function load($id): Ownership
+    {
+        /** @var Ownership $ownership */
+        $ownership = parent::load($id);
```

Comment:
Maybe add an instanceof check here, to make sure it is an Ownership? You can pass an id to it, so you never know when somebody makes a mistake.
You can then drop the @var comment.